src/gettext_boost.cpp

```diff
@@ -11,6 +11,9 @@
 
    See the COPYING file for more details.
 */
+#if defined(_MSC_VER)
+#pragma warning(disable: 4714)
```

Comment:
Wait, why did you move this up here instead of just adding another warning to the other pragma?